MICADO/docs/example_notebooks/MICADO_4_stars.ipynb

```diff
@@ -0,0 +1,256 @@
ig2KU5lml+MR93DeF5YZB3SARcYKqgxoJGXJXV5ZhxVuEXnD81NjYEPegWAeq4E4v3msVNxlm3NNFWmWHn9Ilicl6251bpRSnEtFlNae7nykOzG1GBDtkKrR+kqPzCgYPpyLYaSv3NSZmC/bb7HZGe+oGwgnjfvD7IZtKxBspK951Cl8EcDqddRmQjOp3EeLekRaNbUWoDIfGDVPIG0QW7CDNh4HtjfTPssu4uWpDsP86bT1Dyiv5f3YH3qteQFJWmpdTLvJ22bwoWt8RoVgAi4Ap4HDaj5j6dDs6Lafo5LQKD38oHLyqQ2vbIY6HcJrmMVr106HW4meE2vao7eYJv0unbXzcRiURjNygMlv1mzAdR+qlUtXrcG7NnvKBYQHgRJLT3jzwWQLL7Mh71rCra8cRKSd3v9MkzUrGn+tfZBcygp2/J3y/JsLGMGdDZxKqDgWqP+1kV5kkGNda1TIhO99zqY0r70OhDvD13/Mm5Breo57GYTtRfO++XFMXBJNkoySFWMICmf5mDA2RIFzbGcEz6LEpWhWQMg7UFb9vrQ58pxhmfTL8jOfpNXfsN/PqHrI3rr3/0qq5+xdPBSerUWmaq+lY6tA2M3sJk2+sayiahZw+hND+jFxM3aEX+i3Ap4KOUpm5ETcBGbKG2WQDMjXxt3762lyo0FJE6dFbhxSlOpKYAtN7lhrrrLl83ps3ja6ueUcjZoiNbc6Z6Psze99dlVVHYZ1hWL6XFHKabld2oCZYklMOeU2Xn0GDI1qi5bcY2qUuZ4qeCgUtZwYYk3LG7CsjsDYPYuyUBYs8Z0bIbkC/x52WItnzsZPNnd4cpK55G1gW4NgA2wWBHvRhOXahmKBMbTqEYOexVltZ6Sv3MZLs5ZK8rNRgbBAtvpOlUrNofpRM+LCzVqku/wQLPeWdNbXbrEmHIdVPCCm4O5JkoDYtubqYr9rAOuXSiaaefHXEvL6850Ak8L47ay/05PClxP3GRX5uNiy/ay3bTD1Vwo9nE2S8XgTIU4YorEsrfDRFgqsxYWCtDlib63XBfW7+r+jr336O+9DuQ6ID31oGtCopjfQGWbXkY3adBBxDW+sTGMPWxv3lE2Y8p5sjL5kfV+VLzoa0bvf4vGdnVtAFDjvMpPBKq0xmWBpX050e4wBEFDIHNf3ApW8LaAk1DK2tmdsbjIdDbCkhK/Q3hOIlAFVy8bZTkFksdzZaLG09ZtbJc5H8waZHA/f3fss+M4+4xhhkHPqyCiwBR6jwnc1DJYlsV13NIPftQdA8x1beqVCVzzOw30RtZ6iFpPcY4OzMDsEz2NBWzHayzSiQax3ism5961ALEYc8u0Tp3FtFxF96n1LnkH94J2+8ibQEPFyy7wZRZ3vDBrNUuGcbMulPUlIEZSqiLgn+izz+TVWGoZPnda+1Svjx7wqtBDaTcVqnU6LfvAJWgxqGXnndjDLJ7+yMfTbARQi1mpvQ/uYR5Bb8aFZTzXQRVPG9uqEvhIgrG1HtUaYODmfjsVW2BM6WIfeASwTtUteicAtQlyYx3ULFqndQ4zMi7gEotL0ObNQZvP3ofY+GrxIebma00cgMO7rd+H5A481GwYT+9Xd7lZpBj57oDx1YERdx8+84jWAU9U1Tir0eeJ5uznQD5euysPenxd6DSD7CegynKn4bfpFKk3v0OSoUKz7KoYB5rZGoBaL/sgYEW9moAorWsJwt3eFlSDz7SyuQsgbObi9lA1Yyi9tyy+OIdWO7ax2A1JqjM1Bwiuq4VqAZJBP2EkWrspRjRQjGni9iBX846FdbfGcNV6M60b58BQ+l+ZAgIIFaTJlH51RwHXL4AuYAWqJTefxYVb2QccsBzSz6yc114gAEcm1ezw7CC15tSwLlBFY3Gm5ZS6PCqAFhDleogZHbgJbCc6za5XdN8gdrxGa8dHzfedRXYh5o8gyUEtAFlCqetiAf1m2taf7RS81idBJb1WLbsBquBO7WmBDkCVHeK4OXswUBaFBH+uY1jfn0CpGEzu2foeAb3xwoJLeURrDSiTdgN09W4Z7K+t3SfsMT1emYXSWjh0sGKmV883FoDKJghmtbfE9n4UeH7abnXLQDWzeGU7b36frOPWGoFymcHtO431frROCGT5u5ZpAA7AxstogK+5GBWEGzzPkvDZKtHSvOj22SJm7veWBPDeXAAq1pjvIBVscg6M582/h4QZVktcqFlWMO8AnkHIfHt1jUu8O9zIdlwnwrPMrZCVabmOliwCS+Yo2q5vzWZ9vj6Z66MHvNq4Xdyg9SdQwQMEgDfhuJRgPe9uDxZvSGPZWALFhnEhylfSlcjcLCu9OHsz0obCiHcqfXURyEqyVDDYsnh+uTezAlH6LkswtNl+ENEqMp/sUiOAGuewTg8GubkU2oWLMYKpvzW1X1XfE0oTinHtiDkbHBOgTW3ycluw7ADY3+zUMKM3nh3lg/ga/NHSvMXOQIMgX2BtsBhk7cMeydamAUBdoPRux6JtXkCFtHT1rmYBxedDvSf+vmx2as5hYV4bYOWBaU4Z5fM9V7epJfXGAq15P9tB4GXrQqA9/HcuBNmZctaBqd/Tga7AQ9Xrk4VgZkzrj1nTANlAHq4Eh9anZs/JeSpLKttdEfTIo1kA1jIXBj66d9l8JQFE3nV63W4Ks975fN34TkYfOlxX+bDfFj+tNnFcj64CF8jiWAE8kGVNJzcVXcLfuwLNJQAYXKdvrwRWtwdiqmiN8hpLOI7Z3fbe7Pap9XpAs19VqAfPRxclUYetOZxBNpuZmO153Lq9ZM2FKiKatPPi2rxEAQUVZQ2UZ+oenbVYx4T7ZT7Q0kltf51JqHHM+726TpKB7XnOObBIWso+sCQbBk6ra4lAj5h47SGqxkcHBLJJsxXVWPYSoLMkqKDN8+9+X4qepoPXJiCm7x8B1xbceJTrEoiiL7Dlanu5XUzt6d7nyznBkpYEtneLq4X2SYBthbOD40sDPxw7k9ZnQRTrybkC4Lawk6yqvyPhD
```

Comment:
This should all just work, but apparently there are other things broken and therefore none of the notebooks work, see https://github.com/AstarVienna/irdb/actions/runs/3150371248/jobs/5123073223 .

So it is effectively impossible to review this PR, because of the ScopeSIM (or ScopeSIM_Templates) bugs that need to be fixed first.

My suggestion is to first ensure that all the notebooks run on the dev_master branch, and only then try to add new notebooks.